### ISSUE TITLE
svelte-language-server uses incorrect binary name

### ISSUE THREAD

**Issue by diamondburned** [mattn/vim-lsp-settings#682]
Currently, `svelte-language-server` is set up to search for a `svelte-language-server` binary and will install a `svelte-language-server` wrapper script if the user prompts the plugin to download the LSP:

https://github.com/mattn/vim-lsp-settings/blob/b986f8c10056faf289d692ea816bfc22b780e425/installer/install-svelte-language-server.sh#L5-L6

https://github.com/mattn/vim-lsp-settings/blob/b986f8c10056faf289d692ea816bfc22b780e425/settings/svelte-language-server.vim#L5

This, however, is not what `svelte-language-server` names its LSP server. Instead, `svelte-language-server` installs `svelteserver` to `node_modules/.bin`. As a result, the plugin can never find the actual LSP server binary when it's installed properly.

In other words, the only way to have working Svelte LSP is to install it globally using vim-lsp-settings' supplied shell script instead of using the locally/project-installed `svelteserver` binary.
